front/app/components/admin/AddCollaboratorsModal/index.tsx

```diff
@@ -32,75 +33,83 @@ const AddCollaboratorsModal = ({
   const { formatMessage } = useIntl();
   const { data: appConfiguration } = useAppConfiguration();
   const { data: seats } = useSeats();
+  const [showSuccess, setShowSuccess] = useState(false);
 
   if (!appConfiguration || !seats) return null;
 
   const maximumCollaborators =
     appConfiguration.data.attributes.settings.core.maximum_moderators_number;
   const currentCollaboratorSeats =
     seats.data.attributes.project_moderators_number;
-  const hasReachedLimit =
+  const hasReachedOrIsOverLimit =
     !isNil(maximumCollaborators) &&
     currentCollaboratorSeats >= maximumCollaborators;
-  const buttonText = hasReachedLimit
+  const hasExceededSetSeats =
+    !isNil(maximumCollaborators) &&
+    currentCollaboratorSeats > maximumCollaborators;
+  const buttonText = hasReachedOrIsOverLimit
     ? formatMessage(messages.buyAdditionalSeats, {
         noOfSeats: noOfCollaboratorSeatsToAdd,
       })
     : formatMessage(messages.confirmButtonText);
 
+  const header = !showSuccess ? (
+    <Text color="primary" my="8px" fontSize="l" fontWeight="bold" px="2px">
+      {formatMessage(messages.giveCollaboratorRights)}
+    </Text>
+  ) : undefined;
+
   return (
-    <Modal
-      opened={showModal}
-      close={closeModal}
-      header={
-        <Box px="2px">
-          <Text color="primary" my="8px" fontSize="l" fontWeight="bold">
-            {formatMessage(messages.giveCollaboratorRights)}
-          </Text>
-        </Box>
-      }
-    >
-      <Box display="flex" flexDirection="column" width="100%" p="32px">
-        <Box>
-          <Text color="textPrimary" fontSize="m" my="0px">
-            {hasReachedLimit ? (
-              <FormattedMessage
-                {...messages.reachedLimitText}
-                values={{
-                  noOfSeats: noOfCollaboratorSeatsToAdd,
-                }}
-              />
-            ) : (
-              <FormattedMessage
-                {...messages.confirmMessage}
-                values={{
-                  noOfPeople: noOfCollaboratorSeatsToAdd,
+    <>
+      <Modal opened={showModal} close={closeModal} header={header}>
+        {showSuccess ? (
+          <SeatSetSuccess
+            closeModal={closeModal}
+            hasExceededSetSeats={hasExceededSetSeats}
+            seatType="collaborator"
+          />
+        ) : (
+          <Box display="flex" flexDirection="column" width="100%" p="32px">
+            <Text color="textPrimary" fontSize="m" my="0px">
+              {hasReachedOrIsOverLimit ? (
+                <FormattedMessage
+                  {...messages.hasReachedOrIsOverLimit}
+                  values={{
+                    noOfSeats: noOfCollaboratorSeatsToAdd,
+                  }}
+                />
+              ) : (
+                <FormattedMessage
+                  {...messages.confirmMessage}
+                  values={{
+                    noOfPeople: noOfCollaboratorSeatsToAdd,
+                  }}
+                />
+              )}
```

Comment:
I think this can be simplified since we don't show the modal if we haven't reached the limit? Or it can be done in the specific ticket about not showing the modal in that case.

Reply:
Next ticket